Panto/css/style.css

```diff
@@ -50,18 +54,97 @@
 .hero {
   text-align: center;
   color: var(--color-white);
+  padding-block: 16rem;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 4.4rem;
+}
+
+.hero-text-box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 2.4rem;
+}
+
+.hero-text-box h1 {
+  width: 80%;
+}
+
+.hero-description {
+  width: 45%;
+  font-size: var(--font-size-hero-description);
+  line-height: var(--line-height-hero-footer-p);
+  opacity: 0.8;
+}
+
+.search-bar {
+  position: relative;
+  backdrop-filter: blur(0.3rem);
+  border-radius: 4.2rem;
+  width: 32rem;
 }
 
 .search-bar input[type="search"] {
```

Comment:
what if you add another search input that doesn't share any styles with this selector? use classes instead